packages/suite/src/actions/settings/walletSettingsActions.ts

```diff
@@ -84,3 +86,19 @@ export const getLastUsedFeeLevel = () => (_: Dispatch, getState: GetState) => {
     if (selectedAccount.status !== 'loaded') return;
     return settings.lastUsedFeeLevel[selectedAccount.account.symbol];
 };
+
+export const setBitcoinUnits = (units: PROTO.AmountUnit): WalletSettingsAction => ({
+    type: WALLET_SETTINGS.SET_BITCOIN_UNITS,
+    payload: units,
+});
+
+export const toggleBitcoinUnits = () => (dispatch: Dispatch, getState: GetState) => {
```

Comment:
```suggestion
export const toggleAmountUnit = () => (dispatch: Dispatch, getState: GetState) => {
```
let's keep consistent naming